outputter/database.go

```diff
@@ -0,0 +1,91 @@
+package outputter
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aquasecurity/bench-common/check"
+	"github.com/jinzhu/gorm"
+)
+
+// PSGSQL contains the database connection information
+type PSGSQL struct {
```

Comment:
Does the go-linter not mind this fully-capitalized type name? Also, I suspect there is an extra S here - do you really mean `PgSQL` rather than `PsgSQL`?